### PR TITLE
Continue financial report

### PR DESCRIPTION
**Pull request by mrannanj**
Continue #510

### CODE REVIEW

File: src/game/scenes/arena.c [hunk 1]
```diff
@@ -534,12 +537,26 @@ void arena_har_defeat_hook(int player_id, scene *scene) {
             player_winner->pilot->wins++;
             player_loser->pilot->losses++;
             if(player_id == 1) {
+                fight_stats->winnings = player_loser->pilot->winnings;
+                fight_stats->repair_cost = (1.0f - ((float)winner_har->health / (float)winner_har->health_max)) * 1000;
```
Comment (Vagabond) [inline src/game/scenes/arena.c:541]: Does this vary with difficulty?

Comment (mrannanj) [reply src/game/scenes/arena.c:541]: Eventually probably yes, but not in this implementation.

File: src/game/scenes/arena.c [hunk 2]
```diff
@@ -534,12 +537,26 @@ void arena_har_defeat_hook(int player_id, scene *scene) {
...
+                fight_stats->profit = fight_stats->winnings - fight_stats->repair_cost;
                 player_winner->pilot->rank--;
-                player_winner->pilot->money += player_loser->pilot->winnings;
+                if((int)player_winner->pilot->money + fight_stats->profit < 0) {
+                    player_winner->pilot->money = 0;
```
Comment (Vagabond) [inline src/game/scenes/arena.c:545]: HAR parts also get sold here, right?

Comment (mrannanj) [reply src/game/scenes/arena.c:545]: Not implemented yet.

**Pull request merged**